packages/suite/src/components/settings/SettingsMenu/index.tsx

```diff
@@ -20,12 +19,12 @@ const CloseButtonSticky = styled(CloseButton)<{ isAppNavigationPanelInView?: boo
     ${({ isAppNavigationPanelInView }) =>
         !isAppNavigationPanelInView &&
         css`
-            animation: ${FADE_IN} 0.3s;
+            animation: ${FADE_IN} 0.5s;
             position: fixed;
         `}
 `;
 
-const SettingsMenu = () => {
+export const SettingsMenu: React.FC = () => {
```

Comment:
and done

Reply:
It's better to do TS their job on spoil code with unnecessary types that could be inherited. I really don't want to create our special type, because it will add overhead for people that don't know that and benefits like ensuring return type are not enough. If you return something bad it will throw in place where it's is used.

Let's use just `({...props}: PropsType) => ...`, this is type safe for 98% usages, it's clean because there is no unnecessary code, your PropsType could be used as docs for that component without any implicitly hidden `children` etc.

Reply:
Ok, let's do it like that.